frontend/src/scenes/experiments/ExperimentResult.tsx

```diff
@@ -0,0 +1,228 @@
+import { Col, Progress, Row, Skeleton, Tooltip } from 'antd'
+import { useValues } from 'kea'
+import { ChartDisplayType, FilterType, FunnelVizType, InsightShortId, InsightType } from '~/types'
+import './Experiment.scss'
+import { experimentLogic } from './experimentLogic'
+import { InfoCircleOutlined } from '@ant-design/icons'
+import { FunnelLayout } from 'lib/constants'
+import { capitalizeFirstLetter } from 'lib/utils'
+import { getSeriesColor } from 'lib/colors'
+import { EntityFilterInfo } from 'lib/components/EntityFilterInfo'
+import { NodeKind } from '~/queries/schema'
+import { filtersToQueryNode } from '~/queries/nodes/InsightQuery/utils/filtersToQueryNode'
+import { Query } from '~/queries/Query/Query'
+
+export function ExperimentResult(): JSX.Element {
+    const {
+        experiment,
+        variants,
+        experimentResults,
+        countDataForVariant,
+        exposureCountDataForVariant,
+        experimentInsightType,
+        experimentResultsLoading,
+        conversionRateForVariant,
+        getIndexForVariant,
+        areTrendResultsConfusing,
+        experimentResultCalculationError,
+        sortedExperimentResultVariants,
+        experimentCountPerUserMath,
+    } = useValues(experimentLogic)
+
+    const secondaryColumnSpan = Math.floor(24 / (variants.length + 2)) // +2 for the names column
```

Comment:
This might also be nice as a kea selector:
```
secondaryColumnSpan: [
    (s) => [s.variants],
    (variants): number => Math.floor(24 / (variants.length + 2)) // +2 for the names column
]

It is recomputing every time the component re-renders here, whereas it is cached in the logic until `variants` (upon which it depends) changes